### PR TITLE
Update create-table-azure-sql-data-warehouse.md

### PR DESCRIPTION
**Pull request by CamronBorealis**
Updating invalid link for indexing reference

### CODE REVIEW

**Comment by ktoliver** [comment]
#label:"aq-pr-triaged"

File: docs/t-sql/statements/create-table-azure-sql-data-warehouse.md [hunk 1]
```diff
@@ -329,7 +329,7 @@ To change a rowstore table to a columnstore table, drop all existing indexes on
 
 For more information, see these articles:
 - [Columnstore indexes versioned feature summary](../../relational-databases/indexes/columnstore-indexes-what-s-new.md)
-- [Indexing tables in [!INCLUDE[ssSDW](../../includes/sssdwfull-md.md)]](https://azure.microsoft.com/documentation/articles/sql-data-warehouse-tables-index/)
+- [Indexing tables in [!INCLUDE[ssSDW](../../includes/sssdwfull-md.md)]](https://docs.microsoft.com/en-us/azure/synapse-analytics/sql-data-warehouse/sql-data-warehouse-tables-index)
```
Comment (julieMSFT) [inline docs/t-sql/statements/create-table-azure-sql-data-warehouse.md:332]:
```suggestion
- [Indexing tables in [!INCLUDE[ssSDW](../../includes/sssdwfull-md.md)]](/azure/synapse-analytics/sql-data-warehouse/sql-data-warehouse-tables-index)

```

**Review by julieMSFT — COMMENTED** [review]
Hi @CamronBorealis Thank you for your first contribution to Microsoft Docs. One minor edit to the link to make it site relative. Once you make this update we can merge your change.

**Comment by PRMerger16** [comment]
@julieMSFT : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Comment by julieMSFT** [comment]
#sign-off

**Pull request merged**